src/elements/Volunteers.js

```diff
@@ -13,14 +13,15 @@ const Volunteers = ()  => {
 
   useEffect(() => {
     // get api data here later
+    axios.get("./database.json").then((res) => setVolunteers(res.data.volunteers))
     axios.get("./database.json").then((res) => setSortedVolunteers(res.data.volunteers))
```

Comment:
I know that this was probably a bad clue given by the original template. But what do you think about directly importing the json file via an import statement instead?

That way, we could potentally remove the useEffect hook. Having said that, I'm not too sure why the `filterRef.current.focus()` is there though.

Reply:
I think the `filterRef.current.focus()` line was added in to snap the browswer view to the volunteers section. I believe there was originally the homepage hero section which took up the initial view. I think it has since been removed, so we should be able to remove the useEffect hook without any problems.

Reply:
@davidtaing , Tried importing database.json to the js file. Seems like json file is not being parsed as module, so we get "cannot find module" error. Looks like we are using webpack version 4.29.6, Shall we use json-loader to overcome this ?